### ISSUE TITLE
Setting BlockMacSync doesn't have any effect.

### ISSUE THREAD

**Issue by SebastianGoers** [microsoft/Microsoft365DSC#862]
#### Details of the scenario you tried and the problem that is occurring
Setting BlockMacSync to $true doesn't have any effect.  Even after a configuration has been successfully applied to the tenant DSC will keep reporting a configuration drift.
#### Verbose logs showing the problem
The setting keeps unchanged. There aren't any verbose outputs as the cmdlet is being piped to Out-Null.

#### Suggested solution to the issue
Remove the dependency of an AD group Guid to be set. According to the documentation BlockMacSync should also work w/o an AD group guid:

```
if ($Options.ContainsKey("BlockMacSync") -and $Options.ContainsKey("DomainGuids"))
    {
        Write-Verbose -Message "Updating BlockMacSync {$($Options.BlockMacSync)}"
        Set-PnPTenantSyncClientRestriction -BlockMacSync:$Options.BlockMacSync -DomainGuids $Options.DomainGuids -Enable:$true | Out-Null
    }
    elseif ($Options.ContainsKey("DomainGuids") -and ($Options.ContainsKey("BlockMacSync") -eq $false))
    {
        Write-Verbose -Message "Updating DomainGuids"
        Set-PnPTenantSyncClientRestriction -DomainGuids $Options.DomainGuids -Enable:$true| Out-Null
    }

```
Remove Out-Null so errors can bee seen in the logs.

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
        Set-PnPTenantSyncClientRestriction -BlockMacSync:$Options.BlockMacSync -DomainGuids $Options.DomainGuids -Enable:$true | Out-Null
```
Intersting enough, the Domain GUID are being set, but BlockMacSync and Enable:$true are being disregarded. (Tested results in Admin GUI)

#### Version of the DSC module that was used ('dev' if using current dev branch)
Current production branach (1.20.1021.1).

**Comment by desmay** [comment]
@SebastianGoers Hope to fix this along with other bugs you posted for next release

**Comment by NikCharlebois** [comment]
Closed with #868
